cluster-autoscaler/hack/update-vendor.sh

```diff
@@ -1,229 +1,22 @@
-#env /usr/bin/bash
+#!/usr/bin/env bash
 
```

Comment:
would it be considered bad form to include a `set -x` here?

i always find it a little unsettling when running a shell script that emits no output for long periods of time.

Reply:
Yeah, I'd definitely be happy with that. I think the old script didn't have that flag, but it did output some messages manually. Anyone else have an opinion?